src/posix/platform/trel.cpp

```diff
@@ -440,6 +458,7 @@ void otPlatTrelDisable(otInstance *aInstance)
     trelDnssdStopBrowse();
     trelDnssdRemoveService();
     sEnabled = false;
+    ResetCounters();
```

Comment:
Do we want to reset the counters when TREL is disabled? Or keep it going?
Or can add a new API to allow user reset it?

Also suggest we document the expected behavior in `platform/trel.h` so a platform developer knows what they are expected to write?

Reply:
Now I added cli support and we don't reset the counters when `trel disable`.